lumfn.py

```diff
@@ -37,7 +37,9 @@ def lumfn(dat, Ms=np.arange(-25.5, -15.5, 0.1), Mcol='MCOLOR_0P0'):
 
         ivmax   = 1. / vmax
         ivmax2  = 1. / vmax**2.
-        
+
+        # nsample == 0; set M to mid bin.  Ideally, no nans.
```

Comment:
If this isn't going in (no problem), can we add a #TODO: comment above, that we can grep.